src/adapter/src/catalog.rs

```diff
@@ -6153,28 +6115,7 @@ pub enum Op {
         item: CatalogItem,
         owner_id: RoleId,
     },
-    DropDatabase {
-        id: DatabaseId,
-    },
-    DropSchema {
-        database_id: DatabaseId,
-        schema_id: SchemaId,
-    },
-    DropRole {
-        id: RoleId,
-        name: String,
-    },
-    DropCluster {
-        id: ClusterId,
-    },
-    DropClusterReplica {
-        cluster_id: ClusterId,
-        replica_id: ReplicaId,
-    },
-    /// Unconditionally removes the identified items. It is required that the
-    /// IDs come from the output of `plan_remove`; otherwise consistency rules
```

Comment:
There is no function in the codebase called `plan_remove`, so clearly it's not required for the IDs to come from that function.